app/controllers/api/v1/users_controller.rb

```diff
@@ -3,6 +3,31 @@
 module Api
   module V1
     class UsersController < ApplicationController
+      skip_before_action :authenticate!, only: :create
+
+      def index
+        users = User.all
+        render json: users.as_json(json_options)
+      end
+
+      def create
+        user = User.new(user_params)
+        if user.save
+          render json: user.as_json(json_options)
```

Comment:
You can use a [serializer](https://github.com/jsonapi-serializer/jsonapi-serializer) to show the created user.